### PR TITLE
Gio/screen refactor button

### PR DESCRIPTION
**Pull request by digmel**
Icon component added some refactor in the screen directory & components

### CODE REVIEW

**Comment by digmel** [comment]
Simulator Screen Shot - iPhone 12 - 2021-09-02 at 21 47 25

**Comment by digmel** [comment]
Needs section and screen component, it's just a builerplate

File: src/components/icon/Icon.type.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import {TextStyle, ViewStyle} from 'react-native';
+
+export type TIconProps = {
+  name: string;
+  size?: number;
+  iconContainerStyle?: ViewStyle | object;
```
Comment (lasharela) [inline src/components/icon/Icon.type.ts:6]: object is not needed

File: src/components/icon/Icon.type.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  iconStyle?: TextStyle | object;
```
Comment (lasharela) [inline src/components/icon/Icon.type.ts:7]: same

**Review by lasharela — APPROVED** [review]

**Pull request merged**